src/mbgl/map/live_tile.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef MBGL_MAP_LIVE_TILE
+#define MBGL_MAP_LIVE_TILE
+
+#include <map>
+
+#include <mbgl/map/geometry_tile.hpp>
+
+namespace mbgl {
+
+class LiveTileFeature : public GeometryTileFeature, private util::noncopyable {
```

Comment:
These classes are derived from `util::noncopyable` already so you don't need to re-inherit it here.